_data/accessibility-tests/identifier.yml

```diff
@@ -0,0 +1,103 @@
+title: Identifier
+component_status: pass
+test_items:
+# General tests
+  - summary: Color alone is not used to convey information.
+    summary_additional: When viewing the identifier, you never have to rely on color alone to understand information.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.1
+  - summary: Identifier icons meets color contrast requirements.
+    summary_additional: When you use a color contrast checker to view the identifier, contrast between the icon and adjacent colors is at least 3:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.3
```

Comment:
**Issue (blocking)**: This criterion should be [1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html). We should also move it to numerical order after it's updated 👍 

```suggestion
    wcag_criterion: 1.4.11
```

Reply:
Good catch! 🏄 Updated and re-ordered in [8f528fb](https://github.com/uswds/uswds-site/pull/2754/commits/8f528fb8d52cbd8a893e409514c513f47aa35b34)

Reply:
Good catch! 🏄 Updated and reordered in [8f528fb](https://github.com/uswds/uswds-site/pull/2754/commits/8f528fb8d52cbd8a893e409514c513f47aa35b34)